src/__screenshot_tests__/fixed-footer-screenshot-test.tsx

```diff
@@ -11,3 +11,29 @@ test('Dialog over fixed footer', async () => {
     const image = await page.screenshot();
     expect(image).toMatchImageSnapshot();
 });
+
+test('Fixed footer with relative position when height is smaller than the footer itself', async () => {
+    const page = await openStoryPage({
+        id: 'layout-fixed-footer-layout--more-complex-footer',
+        device: 'MOBILE_IOS_SMALL',
+    });
+
+    const image = await page.screenshot();
+    expect(image).toMatchImageSnapshot();
+
+    await page.setViewport({
+        width: 564,
+        height: 154,
+        deviceScaleFactor: 1,
+        isMobile: true,
+        isLandscape: false,
+        hasTouch: true,
+    });
+
+    const imageOverflow = await page.screenshot({fullPage: false});
+    expect(imageOverflow).toMatchImageSnapshot();
+
+    await (await screen.findByText('Load more text')).evaluate((el) => el.scrollIntoView());
+    const imageOverflowScrolled = await page.screenshot({fullPage: false});
+    expect(imageOverflowScrolled).toMatchImageSnapshot();
```

Comment:
what happened here?
![image](https://user-images.githubusercontent.com/579417/196109777-7dd11742-1cce-4a07-b192-913ddad21fe2.png)

Reply:
I've checked myself on safari, chrome and firefox and the text is well loaded. I guess it's something to do with the dockerized chromium because I can't reproduce it

Reply:
weird, it would be the first time that happens

Reply:
![test](https://user-images.githubusercontent.com/61736059/196675927-db5b94fa-2433-4a67-9ec9-210e373bef9f.jpg)
it's visible even without headless. I'll dig further into this